hack/make-helm-chart.sh

```diff
@@ -25,9 +25,9 @@ TAG="${TAG:?TAG env variable must be specified}"
 HELM_CHART_REPO="us-docker.pkg.dev/online-boutique-ci/charts"
 
 cd helm-chart
-sed -i "s/^appVersion:.*/appVersion: \"${TAG}\"/" Chart.yaml
-sed -i "s/^version:.*/version: ${TAG:1}/" Chart.yaml
+gsed -i "s/^appVersion:.*/appVersion: \"${TAG}\"/" Chart.yaml
+gsed -i "s/^version:.*/version: ${TAG:1}/" Chart.yaml
 helm package .
-helm push onlineboutique-$TAG.tgz oci://$HELM_CHART_REPO
+helm push onlineboutique-${TAG:1}.tgz oci://$HELM_CHART_REPO
```

Comment:
The `TAG` environment variable's value looks like: `v0.5.1`.
Using `${TAG:1}` means that we remove the `v`.

Reply:
Good catch!